imagescan/controller.go

```diff
@@ -433,3 +448,20 @@ func (s *Controller) syncFromRemoteState(ctx context.Context) {
 	}
 	s.log.Infof("images updated from remote state, full_resync=%v, scanned_images=%d", resp.Images.FullResourcesResyncRequired, len(resp.Images.ScannedImages))
 }
+
+func IsImagePending(v *image, now time.Time) bool {
```

Comment:
These funcs IsImagePending and IsImagePrivate can be private as they are not used outside of this pkg.